frontend/spec/features/accessions_spec.rb

```diff
@@ -0,0 +1,571 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rails_helper'
+
+describe 'Accessions', js: true do
+  let(:accession_title) { 'Test accession title' }
+  let(:user) { create_user(repo => ['repository-archivists']) }
+  let(:repo) { create(:repo, repo_code: "accession_test_#{Time.now.to_i}") }
+
+  before(:each) do
+    login_user(user)
+    run_all_indexers
+    select_repository(repo)
+  end
+
+  it 'can create an accession' do
+    click_on('Create')
+    click_on('Accession')
+    fill_in('Title', with: accession_title)
+    fill_in("Identifier", with: "test_#{Time.now}")
+    fill_in("Accession Date", with: "2012-01-01")
+    fill_in("Content Description", with: "Lots of paperclips")
+    fill_in("Condition Description", with: "pristine")
+
+    # Αdd a date
+    click_button('Add Date')
+    select 'Digitized', from: 'Label'
+    select 'Single', from: 'Type'
+    fill_in("Expression", with: "The day before yesterday.")
+
+    # Αdd a second date
+    click_button('Add Date')
+    element = find('#accession_dates__1__label_')
+    element.select 'Digitized'
+    element = find('#accession_dates__1__date_type_')
+    element.select 'Bulk Dates'
+    fill_in 'accession_dates__1__begin_', with: '2021-01-01'
+    fill_in 'accession_dates__1__end_', with: '2021-01-03'
+
+    # Add a rights subrecord
+    click_on('Add Rights Statement')
+    select 'Copyright', from: 'Rights Type'
+    select 'Copyrighted', from: 'Status'
+    fill_in("Start Date", with: "2012-01-01")
+    element = find('#accession_rights_statements_ .subrecord-form-heading .btn:not(.show-all)', match: :first)
```

Comment:
@thimios It looks like maybe this line was from an earlier attempt at finding the 'jurisdiction' dropdown button that line 48 below solves and so was meant to be removed?

Reply:
@brianzelip good catch!